gpii/configs/preferencesServer.production.json

```diff
@@ -12,7 +12,7 @@
             }
         }
     },
-    "includes": [
+    "mergeConfigs": [
         "../node_modules/preferencesServer/configs/development.json",
```

Comment:
Should be "../node_modules/preferencesServer/configs/gpii.preferencesServer.config.development.json"